components/css/content/_contentPage.scss

```diff
@@ -52,11 +52,41 @@
 					width:100%;
 					margin-bottom:20px;
 				}
-				ul{
+				ul, ol {
+					list-style-type: none;
 					li{
 						padding:3px 0;
 					}
 				}
+				ol > li, ul > li {
+					&::before {
+						width: 1em;
+						margin-left: -1em;
+						position: relative;
+						color: $color-campusGold;
+						font-size: calc-toRem(18);
+					}
+				}
+				ul > li {
+					&::before {
+						content: "•";
+						display: inline-block;
+						font-size: calc-toRem(33);
+						line-height: 0;
+						top: 5px;
+						right: -6px;
+					}
+				}
+				ol {
+					counter-reset: li;
```

Comment:
Using `counter-reset` and `conter-increment` is pretty slick. This could break how ordered lists display in older browsers though. Perhaps just wrap these properties in feature queries, `@supports(counter-reset:li)`, so that numbered lists can be displayed properly in older browsers. This would mean that numbers would not be colored in IE, https://caniuse.com/#search=%40support , but that is ok with me.

Reply:
Done. I lumped all the gold styles for both lists in the feature query. Technically I could make the bullets gold regardless of counter support, but for consistency, I opted to have all or nothing. I'll confirm with Garrett.

Reply:
Do you mean the bullets or the numbers? If the bullets are ok regardless of counter support, those don't have to be placed into the feature query.